### PR TITLE
Remove data structures that are added to, but never used.

### PR DESCRIPTION
**Pull request by DanVanAtta**
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/701)

### CODE REVIEW

File: src/games/strategy/triplea/delegate/AirBattle.java [hunk 1]
```diff
@@ -554,9 +554,6 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
 
         @Override
         public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
```
Comment (RoiEXLab) [inline src/games/strategy/triplea/delegate/AirBattle.java]: Is the stack used in other classes implementing IExecutable? If not stack could be removed

**Comment by RoiEXLab** [comment]
No compile Errors, checks are passing this looks good

File: src/games/strategy/triplea/delegate/AirBattle.java (hunk 1 above)
Comment (DanVanAtta) [inline src/games/strategy/triplea/delegate/AirBattle.java]: Part of the interface, but perhaps all implementations do not use `stack`. But, param removal can be relatively complex from a diff point of view - best left to a dedicated PR IMO rather than incur the scope creep here

**Comment by ron-murhammer** [comment]
@DanVanAtta Looks like this needs rebased.

File: src/games/strategy/sound/ClipPlayer.java [hunk 2]
```diff
@@ -117,8 +117,7 @@
 
 
   protected final Map<String, List<URL>> sounds = new HashMap<>();
-  private final Set<String> mutedClips = new HashSet<>();
-  private final Set<String> subFolders = new HashSet<>();
+  private final Set<String> mutedClips = new HashSet<String>();
```
Comment (ron-murhammer) [inline src/games/strategy/sound/ClipPlayer.java]: Bad merge?

File: src/games/strategy/sound/ClipPlayer.java [hunk 3]
```diff
@@ -341,7 +333,7 @@ private URI loadClipPath(final String pathName) {
       resourcePath = SoundProperties.getInstance(resourceLoader).getDefaultEraFolder() + "/" + pathName;
     }
     resourcePath = resourcePath.replace('\\', '/');
-    final List<URL> availableSounds = new ArrayList<>();
+    final List<URL> availableSounds = new ArrayList<URL>();
```
Comment (ron-murhammer) [inline src/games/strategy/sound/ClipPlayer.java]: Bad merge?

File: src/games/strategy/sound/ClipPlayer.java [hunk 4]
```diff
@@ -361,7 +353,7 @@ private URI loadClipPath(final String pathName) {
    *        (URL uses '/', not File.separator or '\')
    */
   protected List<URL> findClipFiles(final String resourceAndPathURL) {
-    final List<URL> availableSounds = new ArrayList<>();
+    final List<URL> availableSounds = new ArrayList<URL>();
```
Comment (ron-murhammer) [inline src/games/strategy/sound/ClipPlayer.java]: Bad merge?

File: test/games/strategy/triplea/delegate/MoveDelegateTest.java [hunk 5]
```diff
@@ -1047,8 +1047,8 @@ public void testReloadTransportAfterRetreatAllied() {
     retreatingLandUnits.addAll(karelia.getUnits().getMatches(Matches.isUnitAllied(russians, m_data)));
     final Integer retreatingLandSizeInt = retreatingLandUnits.size();
     // Get the defending land units that and their number
-    final List<Unit> defendingLandUnits = new ArrayList<>();
     retreatingLandUnits.addAll(karelia.getUnits().getMatches(Matches.isUnitAllied(british, m_data)));
+    final List<Unit> defendingLandUnits = new ArrayList<Unit>();
```
Comment (ron-murhammer) [inline test/games/strategy/triplea/delegate/MoveDelegateTest.java]: Bad merge?

**Comment by ron-murhammer** [comment]
@DanVanAtta Comments added. Mostly looks like bad merge resolutions.

File: test/games/strategy/triplea/delegate/MoveDelegateTest.java (hunk 5 above)
Comment (ron-murhammer) [inline test/games/strategy/triplea/delegate/MoveDelegateTest.java]: Looks like you missed this one.

Comment (DanVanAtta) [inline test/games/strategy/triplea/delegate/MoveDelegateTest.java]: Let's fix it later. I'm okay with the relatively insignificant regression.

Comment (ron-murhammer) [inline test/games/strategy/triplea/delegate/MoveDelegateTest.java]: Why not spend a few minutes while we are looking at this code to make a 1 line commit to fix the merge? This will probably never get fixed if we leave it now since you've already mass moved to diamond notation.

Comment (DanVanAtta) [inline test/games/strategy/triplea/delegate/MoveDelegateTest.java]: - It's not just the few minutes
- We incur yet another round trip on the PR
- It does not matter if it is never fixed. One of these hanging around out of the 2500 removed is not terribly significant
- I disagree we'll never come back to it, I changed them all once, can easily do more passes. That is actually how I fixed them here, by re-running the auto cleanup.
